OWNERS.md

```diff
@@ -0,0 +1 @@
+troy@firstandthird.com
```

Comment:
huh?  I won't want OWNERS.md in every folder.  Couldn't users just click on the commiters?